packages/manager/modules/exchange/src/domain/dkim-autoconfig/domain-dkim-autoconfig.html

```diff
@@ -43,4 +44,81 @@
             </div>
         </div>
     </div>
+
+    <!-- if unconfigured without OVHcloud domain -->
+    <div data-ng-if="ctrl.dkimForNoOvhCloud">
+        <div
+            data-wizard
+            data-wizard-on-finish="ctrl.leaveDkimConfigurator.bind(ctrl)"
+            data-wizard-on-cancel="ctrl.leaveDkimConfigurator.bind(ctrl)"
+            data-wizard-hide-previous-button="true"
+            data-wizard-confirm-button-text=":: 'exchange_tab_domain_diagnostic_dkim_confirm_no_ovhcloud' | translate"
+            data-wizard-next-button-text="ctrl.getNextButtonDependingOnStep() | translate"
+            data-wizard-title="ctrl.getTitleDependingOnStep() | translate"
+        >
+            <div
+                data-wizard-step
+                data-wizard-step-on-load="ctrl.getDkimSelector"
```

Comment:
```suggestion
                data-wizard-step-on-load="ctrl.getDkimSelector.bind(ctrl)"
```
Is this not required ?